tests/unit/mixins/route-test.js

```diff
@@ -308,3 +308,73 @@ test('it uses overridden params when loading more data', assert => {
   assert.ok(model.get('reachedInfinity'), 'Should reach infinity');
 
 });
+
+test('it uses bound params when loading more data', assert => {
+
+  assert.expect(11);
+
+  var RouteObject = Ember.Route.extend(RouteMixin, {
+    model() {
+      return this.infinityModel('item', {perPage: 1, startingPage: 1}, {category: 'feature'});
+    },
+
+    perPageParam: 'testPerPage',
```

Comment:
I don't think these 3 lines are necessary, they are testing the name resolution feature.